core/src/main/java/org/elasticsearch/index/query/FuzzyQueryBuilder.java

```diff
@@ -19,35 +19,54 @@
 
 package org.elasticsearch.index.query;
 
+import org.apache.lucene.index.Term;
+import org.apache.lucene.search.FuzzyQuery;
+import org.apache.lucene.search.MultiTermQuery;
+import org.apache.lucene.search.Query;
+import org.elasticsearch.common.Nullable;
+import org.elasticsearch.common.Strings;
+import org.elasticsearch.common.io.stream.StreamInput;
+import org.elasticsearch.common.io.stream.StreamOutput;
 import org.elasticsearch.common.unit.Fuzziness;
 import org.elasticsearch.common.xcontent.XContentBuilder;
+import org.elasticsearch.index.mapper.MappedFieldType;
+import org.elasticsearch.index.query.support.QueryParsers;
 
 import java.io.IOException;
+import java.util.Objects;
 
 /**
  * A Query that does fuzzy matching for a specific value.
  */
-public class FuzzyQueryBuilder extends MultiTermQueryBuilder implements BoostableQueryBuilder<FuzzyQueryBuilder> {
+public class FuzzyQueryBuilder extends MultiTermQueryBuilder<FuzzyQueryBuilder> implements BoostableQueryBuilder<FuzzyQueryBuilder> {
 
     public static final String NAME = "fuzzy";
 
-    private final String name;
+    public static final Fuzziness DEFAULT_FUZZINESS = Fuzziness.AUTO;
```

Comment:
Personally I like having at least a brief line of JavaDoc on anything that is public that explains it's purpose...